### ISSUE TITLE
trian pascal voc all recall is zeros

### ISSUE THREAD

**Issue by yanmenglu** [AlexeyAB/darknet#207]
I am according the step: How to train (Pascal VOC Data),and I change the following files.
voc.data  

classes= 20
train  = /home/yml/Desktop/darknet_AB/train.txt
valid  = /home/yml/Desktop/darknet_AB/2007_test.txt
names = data/voc.names
backup = /home/yml/Desktop/darknet_AB/backup

yolo-voc.2.0.cfg

[net]
batch=64
subdivisions=8
height=416
width=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.0001
max_batches = 45000
policy=steps
steps=100,25000,35000
scales=10,.1,.1

use the command:
 ./darknet  detector train  cfg/voc.data  cfg/yolo-voc.2.0.cfg   darknet19_448.conv.23

and in the step 26000, the  average loss  avg no longer decreases.I stop the training. 

and  the output in the command:

![image](https://user-images.githubusercontent.com/23131833/30736928-3f1bb014-9fb7-11e7-8109-04bfc3398d5a.png)

,but when I  want to know which weights  is best.
I used in the commond:  
./darknet   detector  recall cfg/voc.data  cfg/yolo-voc.cfg    backup/yolo-voc_26000.weights 
./darknet   detector  recall cfg/voc.data  cfg/yolo-voc.cfg    backup/yolo-voc_24000.weights 
./darknet   detector  recall cfg/voc.data  cfg/yolo-voc.cfg    backup/yolo-voc_10000.weights 

but the result all is : 
![image](https://user-images.githubusercontent.com/23131833/30737013-9f29606e-9fb7-11e7-9a83-88bd3dd6cef9.png)

I need some help ,please help me.
is there something wrong in my training ?

**Comment by AlexeyAB** [comment]
> use the command:
>  ./darknet  detector train  cfg/voc.data  cfg/yolo-voc.2.0.cfg   darknet19_448.conv.23

There you use `yolo-voc.2.0.cfg`

> and  the output in the command:
> 
> ![image](https://user-images.githubusercontent.com/23131833/30736928-3f1bb014-9fb7-11e7-8109-04bfc3398d5a.png)

But there you use `yolo-voc.cfg`

**Comment by yanmenglu** [comment]
ok ,thank you ,I will check the training files again.

**Comment by yanmenglu** [comment]
@AlexeyAB ,the  problem have solved. The wrong is when I choice the best weights, I should use the same  cfg file as when I training.
